README.md

```diff
@@ -107,9 +107,9 @@ Decompression timings are closely matched among `gz`, `xz`, and `zstd` compressi
 OBS Source Services can run in one of several modes as shown in
 [OBS Documentation: Using Source Services: Modes of Services](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.source_service.html#sec.obs.sserv.mode).
 
-Currently the recommended mode is `disabled` (aliased as `manual`) which implies:
+Currently the recommended mode is `localonly` which implies:
 
-- The service is run locally via explicit CLI call `osc service disabledrun`
+- The service is run locally via explicit CLI call `osc service localrun`
```

Comment:
this would no longer be the case with localrun iirc. I think we should advertise manualrun.

Reply:
Afaik it should just be:
osc service run
run         r  run defined services with modes "trylocal", "localonly", or no mode set locally,
                   may take an optional parameter to run only a specified source service. In case
                   parameters exist for this one in _service file they are used.